### ISSUE TITLE
[Attendee Tracking] Submitter does not appear in attendee search results

### ISSUE THREAD

**Issue by Julesssss** [Expensify/App#48695]
### Problem

Submitters do not initially appear in the attendee search results. After deselection they do appear in results. 

![Screenshot 2024-09-06 at 11 20 40](https://github.com/user-attachments/assets/b9508554-1eb3-4923-9811-c49ce18c1f25) ![Screenshot 2024-09-06 at 11 20 45](https://github.com/user-attachments/assets/438d20af-a020-4817-91bc-ec1c68fe1ceb)

https://github.com/user-attachments/assets/deb32c7c-35f5-4de6-bb0e-4a79425c5b31

### Solution 

Resolve. _Located during initial testing of [this PR](https://github.com/Expensify/App/issues/47171)_

**Comment by Julesssss** [comment]
No design required, this is a bug for a new feature,

**Comment by muttmuure** [comment]
Should this be EXTERNAL @Julesssss?

**Comment by Julesssss** [comment]
> Should this be EXTERNAL @Julesssss?

@muttmuure nah, we're working with an agency on this feature 👍

**Comment by zfurtak** [comment]
As Kuba is now working on [Attendee on editing request issue](https://github.com/Expensify/App/issues/47173) I will take care of this bug 😊

**Comment by ZhenjaHorbach** [comment]
@Julesssss 
Can I be appointed as a reviewer here ?

**Comment by zfurtak** [comment]
@Julesssss @ZhenjaHorbach 
I fixed this bug and right now it looks like below:
Is it okay or should I change something?
<img width="200" alt="Screenshot 2024-09-11 at 16 36 07" src="https://github.com/user-attachments/assets/b174e949-c7de-4b64-9682-679a2fbc693d"> <img width="200" alt="Screenshot 2024-09-11 at 16 36 16" src="https://github.com/user-attachments/assets/bc740b13-c696-4f7d-98b3-969cbfe328a7">

**Comment by ZhenjaHorbach** [comment]
> @Julesssss @ZhenjaHorbach I fixed this bug and right now it looks like below: Is it okay or should I change something? <img alt="Screenshot 2024-09-11 at 16 36 07" width="200" src="https://private-user-images.githubusercontent.com/92310164/366526665-b174e949-c7de-4b64-9682-679a2fbc693d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYwNjk1NDMsIm5iZiI6MTcyNjA2OTI0MywicGF0aCI6Ii85MjMxMDE2NC8zNjY1MjY2NjUtYjE3NGU5NDktYzdkZS00YjY0LTk2ODItNjc5YTJmYmM2OTNkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTExVDE1NDA0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMwZjg3MGE5ZTk2YWY3NjZmNGVjYjZiNDk2ZDQ0ODA0YmU4OGM2YTRlMjg3Y2U3NDIxYTlmMGI3NDk4MzcwMzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.xOgpNLj6P1zAbukVt1VguaPEPHBaHCbv_es5Y2TUeOU"> <img alt="Screenshot 2024-09-11 at 16 36 16" width="200" src="https://private-user-images.githubusercontent.com/92310164/366526674-bc740b13-c696-4f7d-98b3-969cbfe328a7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYwNjk1NDMsIm5iZiI6MTcyNjA2OTI0MywicGF0aCI6Ii85MjMxMDE2NC8zNjY1MjY2NzQtYmM3NDBiMTMtYzY5Ni00ZjdkLTk4YjMtOTY5Y2JmZTMyOGE3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTExVDE1NDA0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI3YjU5MTNjNDI5OThkOWFkZjg0YWVlMmFkODcwZGQ2NDVmM2EzOTEwOGM4NzI2YTA3ZDI5MGQ4MDg1MzBkMTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qZFNsgAlsmi1UE75h701ZBeK48lblS1ygVY3JixTEKg">

Looks good at first !
But I think it would be best to test PR 😃

And as far as I remember submitter must be in recently

**Comment by zfurtak** [comment]
> And as far as I remember submitter must be in recently

oh okay! that's why I wanted to make sure 😅
@Julesssss can you confirm that to be 100% sure?

**Comment by ZhenjaHorbach** [comment]
> [Tracking] [External] Attendee Tracking project #44725


Here is the design team's opinion of where the submitter should be

https://github.com/Expensify/App/issues/47171#issuecomment-2325391791

**Comment by zfurtak** [comment]
Thank you, I'm changing that 😊

**Comment by Julesssss** [comment]
> And as far as I remember submitter must be in recents (If this is not a separate issue)

I agree submitter needs to show in recent when selecting attendees, though I'm not sure that is necessary to show the subsections when we are showing the filtered search results 🤔

**Comment by zfurtak** [comment]
It works like this in different parts of the app 🤔

**Comment by ZhenjaHorbach** [comment]
Yep 
For example in start chat during search we have subsections

<img width="371" alt="Снимок экрана 2024-09-11 в 20 09 23" src="https://github.com/user-attachments/assets/318929b9-ec67-4a78-918c-3cce254e9695">

**Comment by Julesssss** [comment]
Ah great, ignore me! For some reason I was imagining that 'Recents' would sit underneath the  'Contacts'
